src/middleware/require_user_agent.rs

```diff
@@ -19,7 +19,8 @@ impl AroundMiddleware for RequireUserAgent {
 
 impl Handler for RequireUserAgent {
     fn call(&self, req: &mut dyn RequestExt) -> AfterResult {
-        let has_user_agent = request_header(req, header::USER_AGENT) != "";
+        let agent = request_header(req, header::USER_AGENT);
+        let has_user_agent = agent != "" && agent != "Amazon CloudFront";
```

Comment:
Can we put this into an environment variable?

Reply:
@pietroalbini I've added a commit moving the string into an environment variable. If the variable is unset, it defaults to our current CDN. The CDN check and be disabled by setting `WEB_CDN_USER_AGENT` to the empty string.